tests/scripts/test-ci.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+function push() {
+    arm64_Master_Image=$(docker images --format "{{.Repository}}:{{.Tag}}" | grep "arm64:latest" | head -n 1)
+    docker tag "$arm64_Master_Image" quay.io/rook/rook-arm64:master
+    docker push quay.io/rook/rook-arm64:master
+    docker tag "$arm64_Master_Image" ghcr.io/rook/rook-arm64:master
+    docker push ghcr.io/rook/rook-arm64:master
+
+    arm64_Release_Image=$(docker images --format "{{.Repository}}:{{.Tag}}" | grep -v "arm64:latest" | grep -v "arm64:master" | grep arm64 | head -n 1)
+    tag_arm64_Release_Image="$(echo "$arm64_Release_Image" | cut -d ":" -f 2)"
```

Comment:
Could you factor out some sub-functions? There are a lot of repetitive commands that could be simplified.

Reply:
thanks for suggestion since script is removed, this is not valid anymore.